main.go

```diff
@@ -59,17 +59,22 @@ func main() {
 		log.Fatal(err)
 	}
 
-	if env.Domain() == "localhost" {
-		log.Fatal(`Domain "localhost" is unsupported; use "127.0.0.1" instead.`)
-	}
+	/*
+		Using 127.0.0.1 is not possible with TLS. Localhost on the underhand works
+
+		if env.Domain() == "localhost" {
+			log.Fatal(`Domain "localhost" is unsupported; use "127.0.0.1" instead.`)
+	}*/
 
 	var srvFn web.ServerType
+	domain := env.Domain()
 	if *dev {
 		srvFn = web.NewDevelopmentServer
+		domain = fmt.Sprintf("https://%s", domain)
 	} else {
 		srvFn = web.NewProductionServer
 	}
-	log.Printf("Starting QuickFeed on %s%s", env.Domain(), *httpAddr)
+	log.Printf("Starting QuickFeed on %s%s", domain, *httpAddr)
```

Comment:
We should print the full URL also for production, so please remove the lines above, and do this:
```go
log.Printf("Starting QuickFeed on https://%s%s", env.Domain(), *httpAddr)
```
Maybe we don't need to print the `httpAddr` if it is `:443` since that's the default `https` port.